src/OpenTelemetry.AutoInstrumentation.Logging/Properties/AssemblyInfo.NetFramework.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="AssemblyInfo.NetFramework.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETFRAMEWORK
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+[assembly: AssemblyKeyFile("keypair.snk")]
```

Comment:
Can you check if you can move move `keypair.snk` to `src` and change this like to:


```suggestion
[assembly: AssemblyKeyFile("..\\keypair.snk")]
```

The same should be then done in `src\OpenTelemetry.AutoInstrumentation.Logging\Properties\AssemblyInfo.NetFramework.cs`

Reference: https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assemblykeyfileattribute.-ctor?view=net-7.0